src/locales/de/translations.json

```diff
@@ -61,24 +61,46 @@
       "description": "Wir suchen Entwickler:innen aus Leidenschaft! Schaue Dir unsere offenen Stellen an. Wir freuen uns auf Deine Bewerbung."
     },
     "thank": "Danke für deine Bewerbung!",
-    "email-confirmation": "Wir haben dir eine E-Mail zur Bestätigung geschickt und melden uns bei dir."
+    "email-confirmation": "Wir haben dir eine E-Mail zur Bestätigung geschickt und melden uns bei dir.",
+    "leadbox": {
+      "title": "Deine Stelle nicht dabei?",
+      "subtitle": "Initiativbewerbung",
+      "text": "Georgios Kaleadis, HR Manager",
+      "mail": "georgios.kaleadis@satellytes.com"
+    }
   },
   "office": {
     "title": "Büro",
     "heading": "Unser Büro",
     "subheading": "Fühl dich wohl bei uns. Ein kleiner Eindruck von unserem Büro in der Sendlinger Straße im Herzen Münchens.",
-    "link": "<0>Wenn du mit uns in diesem Office arbeiten möchtest, dann schau dir doch unsere <1>offenen Stellen</1> an.</0>"
+    "link": "<0>Wenn du mit uns in diesem Office arbeiten möchtest, dann schau dir doch unsere <1>offenen Stellen</1> an.</0>",
+    "leadbox": {
+      "title": "Interesse mit und bei uns zu arbeiten?",
+      "link": "Karriere"
+    }
   },
   "services": {
     "title": "Leistungen",
-    "subheading": "Satellytes – das sind ausschließlich leidenschaftliche Entwickler:innen und Designer:innen."
+    "subheading": "Satellytes – das sind ausschließlich leidenschaftliche Entwickler:innen und Designer:innen.",
+    "leadbox": {
+      "title": "Haben Sie Interesse bekommen?",
+      "subtitle": "Projektanfragen",
+      "text": "Gholam Abdol, Geschäftsführer",
+      "mail": "gholam.abdol@satellytes.com"
+    }
```

Comment:
👍 @abdolg are you good with the public email here for inquiries ?

Reply:
I spoke to Gholam and we should not expose personal emails yet. Let's stick to `info@satellytes` for anything related to gholam.